src/ui/theme/index.ts

```diff
@@ -20,7 +22,9 @@ import { Dict } from './types';
 export function getTheme(colorMode: ColorMode = 'day'): Dict<unknown> {
   return extendTheme(
     {
+      breakpoints,
```

Comment:
We shouldn't need to place this here since we are already extending from the nypl base theme. So we shouldn't need to redefine the breakpoints from the base theme